Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:21.04
+
+RUN apt update && apt install -y ca-certificates
+
+workdir /basilisk
+
+RUN useradd -m -u 1000 -U -s /bin/sh -d /basilisk basilisk && \
+    mkdir -p /basilisk/.local/share/basilisk && \
+    ln -s /basilisk/.local/share/basilisk /data && \
+    chown -R basilisk:basilisk /basilisk
+    
+USER basilisk
+ADD ./basilisk /basilisk/testing-basilisk
+
+EXPOSE 30333 9933 9944
+
+VOLUME ["/data"]
+
+entrypoint [ "/basilisk/testing-basilisk" ]
```

Comment:
ENTRYPOINT, but i prefer CMD instead if possible

Reply:
note that with entrypoint you cannot use basilisk, you can only execute testing basilisk

Reply:
Why do you prefer CMD? You can override both. Entrypoint is easier to be used by para registration script. If you change it to CMD, don't forget to update para. registration script.

Reply:
let's keep ENTRYPOINT for now

Reply:
ok

Reply:
@martinfridrich can the normal basilisk be the default entrypoint then at least ? :)

Reply:
@lumir-mrkva sure, we can make special case for basilisk in para reg. srcipt. Basilisk will be better entrypoint if this images will be public. I just wanted to have unified way to call subcmds in docker images by script. This change have to  be reflected into para reg. script